packages/utilities/src/guards/PermissionGuard/PermissionGuard.ts

```diff
@@ -6,37 +6,45 @@ import type {
   MessageComponentInteraction,
   PermissionString,
 } from "discord.js";
-import { GuildMember } from "discord.js";
+import { GuildMember, MessageEmbed } from "discord.js";
 import type { Client, GuardFunction, Next } from "discordx";
 import { SimpleCommandMessage } from "discordx";
 
-import type { PermissionHandlerInteraction } from "./types";
+import type {
+  PermissionGuardOptions,
+  PermissionHandlerInteraction,
+} from "./types";
 
 /**
  * Set an array of permissions that this command is allowed to use, this is useful for global commands that can not use the `@Permission` decorator, until Permissions 2v is out
  * @param permissions - an array of Permissions or a function to resolve the permissions
- * @param messageIfNotAllowed - message to post when the member using this command does not have the correct permissions, defaults to "No permissions"
+ * @param options - Options for the guard reply. Available options are: `content` (string or MessageEmbed) and `ephemeral` (boolean) defaults to `content`: "No permissions" and `ephemeral`: false
```

Comment:
instead of ``No permissions`` print the required permissions. something like

``you need x, x, x permission for this command``